content/grants/app-engram.md

```diff
@@ -0,0 +1,118 @@
++++
+title = "Engram: a collaborative knowledge managment app"
+date = "2022-03-25"
+
+[taxonomies]
+grant_type = [ "Bounty" ]
+grant_category = [ "App Dev" ]
+
+[extra]
+image = ""
+description = "Engram is a knowledge management application that allows individuals or teams to build rich note documents."
+reward = "$70,000 in WSTR"
+assignee = ""
+id = "B0155"
+champion = "Holium"
+completed = false
+work_request_link = "
+https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0155&prefill_Grant+Name=Engram"
```

Comment:
```suggestion

```

Reply:
You were a little too literal with my comment. :)

Reply:
What else should I do?

Reply:
make sure it is all one line with the link between the quotes

Reply:
fixed